index.html

```diff
@@ -9421,42 +9605,47 @@
               was created by the local or remote peer. When the <code>reset()</code>
               method is called, the user agent <em class="rfc2119" title="MUST">MUST</em>
               run the following steps:</p>
-              <p>1. Let <var>stream</var> be the <code><a>RTCQuicStream</a></code> object
-              which is about to be reset.</p>
-              <p>2. If <var>stream</var>'s  <a>[[\QuicStreamState]]</a> is <code>new</code>,
-              set <var>stream</var>'s  <a>[[\QuicStreamState]]</a> to <code>closed</code>,
-              and abort these steps.</p>
-              <p>3. If <var>stream</var>'s  <a>[[\QuicStreamState]]</a> slot is <code>closed</code>,
-              then abort these steps.</p>
-              <p>4. Set <var>stream</var>'s <a>[[\Writeable]]</a> and <a>[[\Readable]]</a> slots to <code>false</code>.</p>
-              <p>5. Set <var>stream</var>'s <a>[[\QuicStreamState]]</a> slot to <code>closing</code>.</p>
-              <p>6. If the closing procedure has not started yet, start it by sending a RST_STREAM
-              frame.</p>
+              <ol>
+                <li>Let <var>stream</var> be the <code><a>RTCQuicStream</a></code> object
+                which is about to be reset.</li>
+                <li>If <var>stream</var>'s  <a>[[\QuicStreamState]]</a> is <code>new</code>,
+                set <var>stream</var>'s  <a>[[\QuicStreamState]]</a> to <code>closed</code>,
+                and abort these steps.</li>
+                <li>If <var>stream</var>'s  <a>[[\QuicStreamState]]</a> slot is <code>closed</code>,
+                then abort these steps.</li>
+                <li>Set <var>stream</var>'s <a>[[\Writeable]]</a> and <a>[[\Readable]]</a>
+                slots to <code>false</code>.</li>
+                <li>Set <var>stream</var>'s <a>[[\QuicStreamState]]</a> slot to <code>closing</code>.</li>
+                <li>If the closing procedure has not started yet, start it by sending a RST_STREAM
+                frame.</li>
+              </ol>
               <div>
                 <em>No parameters.</em>
               </div>
               <div>
                 <em>Return type:</em> <code>void</code>
               </div>
             </dd>
-            <dt><dfn><code>write</code></dfn></dt>
+           <dt><dfn><code>waitForReadable</code></dfn></dt>
             <dd>
-              <p>When the <code>write</code> method is called, the <a>user agent</a> MUST run the
-              following steps:</p>
+              <p><code>waitForReadable</code> allows the read buffer to
+              grow, resolving the promise when the data queued in the
```

Comment:
I think only setTargetReadBufferedAmount can make the read buffer grow, not waitForReadable. I don't think calling waitForReadable itself allows the read buffer to grow; it just waits.

Reply:
Agree that "allows the read buffer to grow" makes no sense.

Reply:
My assumption was that setTargetReadBufferedAmount was used to manipulate the maximum receive window.  Did the description of its operation make sense?